### PR TITLE
Dime project

### PR DESCRIPTION
**Pull request by SandyRogers**

### CODE REVIEW

File: src/notebooks/R Examples/dimme_examples.ipynb [hunk 1]
```diff
@@ -0,0 +1,1113 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9101d1d0-5745-4f81-bb50-29ef7962887b",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "magnify_subject = all_metadata %>% select('sample_sample-name')\n",
+    "head(magnify_subject)"
```
Comment (SandyRogers) [inline src/notebooks/R Examples/dimme_examples.ipynb:566]:
```suggestion
    "head(mgnify_subject)"
```

File: src/notebooks/R Examples/dimme_examples.ipynb [hunk 2]
```diff
@@ -0,0 +1,1113 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c35758b2-1d57-4449-bc91-5a96a4041108",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "magnify_subject$mgnify_accession <- rownames(magnify_subject)\n",
+    "rownames(magnify_subject) <- NULL"
```
Comment (SandyRogers) [inline src/notebooks/R Examples/dimme_examples.ipynb:592]:
```suggestion
    "mgnify_subject$mgnify_accession <- rownames(mgnify_subject)\n",
    "rownames(mgnify_subject) <- NULL"
```

File: src/notebooks/R Examples/dimme_examples.ipynb [hunk 3]
```diff
@@ -0,0 +1,1113 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a66642dc-4c81-48b8-b6ce-a220a9ed4364",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "head(magnify_subject)"
```
Comment (SandyRogers) [inline src/notebooks/R Examples/dimme_examples.ipynb:604]:
```suggestion
    "head(mgnify_subject)"
```

File: src/notebooks/R Examples/dimme_examples.ipynb [hunk 4]
```diff
@@ -0,0 +1,1113 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8ad8ced4-a2f8-4b12-9a2e-766dcb6bddbb",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "ids_table = merge(start_group, magnify_subject, by.x = \"SubjectID\", by.y = \"sample_sample-name\", all.x = TRUE)"
```
Comment (SandyRogers) [inline src/notebooks/R Examples/dimme_examples.ipynb:624]:
```suggestion
    "ids_table = merge(start_group, mgnify_subject, by.x = \"SubjectID\", by.y = \"sample_sample-name\", all.x = TRUE)"
```

Comment (Ales-ibt) [reply src/notebooks/R Examples/dimme_examples.ipynb:624]: All typos solved! Thanks!

**Comment by SandyRogers** [comment]
@Ales-ibt is it still desirable to merge this notebook onto the public server, or does it rely on an embargoed dataset?

**Comment by SandyRogers** [comment]
> I don't think we should merge the branch. I can store the notebook locally in case the collaborators want it.

Thanks Ales. I'll close this PR but leave the branch on here for future / making public if and when the corresponding data are.

**Pull request closed without merging**